src/tools/add_entity.cc

```diff
@@ -100,39 +100,44 @@ struct add_entity_tool : tool {
         ch->entities.push_back(e);
     }
 
+    // press to rotate once, hold to rotate continuously
     void alt_use(raycast_info *rc) override {
-        do {
-            entity_name_index++;
-            if (entity_name_index >= entity_names.size()) {
-                entity_name_index = 0;
-            }
-        } while (!entity_stubs[entity_names[entity_name_index]].get_component<placeable_component_stub>());
-
         auto rotate = get_rotate();
-        if (rotate == 0) {
-            cur_rotate = 0;
-        }
-        else {
-            cur_rotate += rotate / 2;
-            cur_rotate = cur_rotate - (cur_rotate % rotate);
-        }
+
+        cur_rotate -= rotate;
```

Comment:
Keep rotation positive, negate when producing the matrix?

Reply:
hmm, yeah, could do

Reply:
done